docs/src/main/asciidoc/security-jpa-concept.adoc

```diff
@@ -0,0 +1,180 @@
+[id="security-jpa-concept"]
+= Quarkus Security with JPA
+include::_attributes.adoc[]
+:categories: security,jpa
```

Comment:
Is this a good topic to help a developer get started with Quarkus? If so: 

```suggestion
:categories: security,jpa,getting-started
```

Reply:
@sberyozkin WDYT?

Reply:
Hi @michelle-purcell @MichalMaler, on its own is a fairly technical concept, it is not really something that new users should be forwarded to when starting with Quarkus. It does make sense to show JPA in the getting started with quarkus security (Basic plus JPA) just because saving the users info in DB is better than directly in application.properties, but IMHO on its own it is not a getting started material :-). Perhaps at a later stage when a reactive JPA gets supported we can consider how to make it move visible, thanks

Reply:
@sberyozkin - Thanks for explaining. Sounds good 👍